src/applications/mhv/landing-page/components/LandingPage.jsx

```diff
@@ -14,7 +18,7 @@ const LandingPage = ({ data = null }) => {
     >
       <div className="vads-l-grid-container large-screen:vads-u-padding-x--0">
         <HeaderLayout />
-        <CardLayout data={cards} />
+        {hasRecord ? <CardLayout data={cards} /> : <NoHealthAlert />}
```

Comment:
Some name change suggestions for clarity: 

```
const showCardLayout = useSelector(hasHealthData);
//...
{showCardLayout ? <CardLayout data={cards} /> : <NoHealthDataAlert />}
```

Reply:
Done